acceptance/analyzer_test.go

```diff
@@ -364,32 +372,37 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 					})
 
 					when("no auth registry", func() {
+						var noAuthRegCacheImage string
+
 						it.Before(func() {
 							metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "cache_image_metadata.json"), platform.CacheMetadata{})
-							cacheImage, cacheAuthConfig = buildNoAuthRegistryImage(
+							cacheImageName := "some-cache-image-" + h.RandString(10)
+							authRegCacheImage, _ = buildRegistryImage(
 								t,
-								"some-cache-image-"+h.RandString(10),
+								cacheImageName,
 								filepath.Join("testdata", "analyzer", "cache-image"),
-								"--build-arg", "fromImage="+variables.ContainerBaseImage,
+								"--build-arg", "fromImage="+vh.ContainerBaseImage(),
 								"--build-arg", "metadata="+metadata,
 							)
+
+							noAuthRegCacheImage = noAuthRegistry.RepoName(cacheImageName)
```

Comment:
I found myself scratching my head wondering "when do we push this image" but now I understand (I think) this works because we're using the shared storage volume. I feel like we might want to document this in a comment somewhere...